dos_connect/server/elasticsearch_backend.py

```diff
@@ -126,3 +126,22 @@ def delete(properties, index='data_objects'):
         clauses.append('+{}:"{}"'.format(k, v))
     s = s.query("query_string", query=' '.join(clauses))
     s.delete()
+
+
+def metrics(indexes=['data_objects', 'data_bundles']):
+    """
+    return document counts
+    """
+    def _count(index):
+        try:
+            s = Search(using=client, index=index, doc_type=index[:-1])
+            return s.count()
+        except Exception as e:
+            log.error('error getting count of documents in {}'.format(index))
+            log.exception(e)
+            return 0
```

Comment:
https://prometheus.io/docs/instrumenting/writing_exporters/#failed-scrapes

Not sure, but I think you want to raise an error in this case, or perhaps don't emit a metric. I think by returning zero, you're hiding the error from Prometheus and telling it the count is legitimately zero.

Reply:
Good point.  There is an edge case (no documents written yet to a blank db).  I've accounted for that.